### DISCUSSION TITLE
Nouveau mini event le peintre

### DISCUSSION THREAD

**Discussion by BastLast** [Crownicles/Crownicles#2353] [Nouvelles idées]
Le joueur rencontre un mec qui fais des gravures ou des statues ou un truc du genre et qui vend ses œuvres (rareté minimale et seulement dispo a un endroit de la map) 

C'est assez cher mais si le joueur paye il obtient un item de rareté 9 (oeuvre) chaque œuvre est unique et ne peut pas être obtenue par un autre joueur. 
(faudra faire gaffe d'ailleurs que 2 personnes puissent pas tomber sur le mini event en meme temps) 

Une œuvre a une grande valeur, elle vaut cher à l'achat. Genre 15 à 100k argent et + (en fonction des œuvres (chaque œuvre a une valeur de base) 

Les joueurs peuvent revendre une œuvre avec la commande / sell, les valeur des œuvres varient selon une fonction pseudo random ( comme l'argent du roi ) 

Si une œuvre est vendue par un joueur , elle disparaît pour toujours personne ne pourra l'obtenir. 

Si il n'y a plus d'œuvre en jeu (parce que yen aura pas à linfini) le mini event apparaît toujours mais le mec affiche des infos sur une des œuvres disparues (vendues) 

Les oeuvres occupent un slot d'inventaire dans la catégorie objet

Chaque oeuvre a un nom

Chaque oeuvre est associée à une image qui peut être affichée grâce à une commande dédié ou une réaction sojs l'inventaire ou un truc dans le genre. 

Les œuvres n'ont aucun effet dans l'aventure. 

Exemple :

Le chat démon
Oeuvre
![20231221_094107.jpg](https://github.com/DraftBot-A-Discord-Adventure/DraftBot/assets/38099970/c432f35c-01eb-4c8e-9e8a-4e4d742ca808)

**Comment by BastLast** [comment]
Les photos des œuvres seront des photos réelles de chat ou de trucs comme ca qu'on décidera d'ajouter au bot. À voir comment on intégre la communauté dans ce process :)

**Reply by Voisin1234** [reply]
En tout franchise je trouve sa chelou mais pourquoi pas . Après ça va coûter très cher et ducoup ça va surtout être pour les gros joueur qui on des sous à mettre à la poubelle. Mais je pence que idée peux être améliorer. Et perso si c'est juste des chats j'achète déjà pas le calendrier de la poste alors 😁
